src/main/java/ch/fmi/ContourPolygonBitmaskToText.java

```diff
@@ -0,0 +1,69 @@
+/*-
+ * #%L
+ * Plug-in to convert a 2D binary to a polygon geometry serialized to
+ * well-known text (wkt).
+ * %%
+ * @author Charismatic Claire
+ * %%
```

Comment:
We usually generate the license headers by running [`mvn license:update-file-header`](https://www.mojohaus.org/license-maven-plugin/update-file-header-mojo.html) before each release, which will replace the copyright statement by the content from the `pom.xml`. I hope you're ok if this information gets replaced? The Friedrich Miescher Institute (FMI) legally is the copyright owner of this software, as it is being developed as part of my work at FMI.

Your *authorship* is of course unaffected, and will be visible from the git history. We can also add a *class javadoc* containing an `@author` tag. Would you be ok adding your real name in such a tag?

Reply:
Yeah, I changed that accordingly. I haven't decided about my real name being put somewhere in the code. So I think I just skip that part for now.